gprofiler/profilers/java.py

```diff
@@ -570,6 +574,9 @@ def _copy_libap(self) -> None:
                 os.chmod(
                     libap_resource, 0o755
                 )  # make it accessible for all; needed with PyInstaller, which extracts files as 0700
+                os.chmod(
```

Comment:
This doesn't make much sense to me. Why is asprof needed in the process context? It is not loaded by the process / invoked by it. We invoke it from gprofiler context.

Reply:
👍 
That's right. It appeared here as an artifact of early experiments with ap binary launcher.